assets/scss/partials/components/_notice.scss

```
@@ -0,0 +1,44 @@
+.notice {
+  display:  flex;
+  flex-direction: column;
+
+  @include themed() {
+    background-color: mix(t('info'), t('accent'), 40%);
+  }
+
+  &--update {
+
+    @include themed() {
+      background-color: mix(t('success'), t('accent'), 40%);
+    }
+  }
+  &--warning {
+
+    @include themed() {
+      background-color: mix(t('danger'), t('accent'), 40%);
+    }
+  }
+
+  &__title {
+    background-color: t('info');
+    align-self: flex-end;
+    font-weight: 300;
+    letter-spacing: 0.025em;
+    padding: 0.2rem 0.2rem 0.2rem 0.2rem;
```

I'd give this a little more room to breath on the x-axis:

```suggestion
    padding: 0.2rem 0.5rem 0.2rem 0.5rem;
```